gst.so/src/gst_worker.cpp

```diff
@@ -489,6 +498,38 @@ bool GSTWorker::do_gst_stress_test(int *error, std::string *err_description) {
       }
     }
 
+    if (self_check || accu_check) {
+
+      if (error_inject) {
+        gpu_blas->set_gemm_error(error_freq, error_count);
+      }
+
+      double self_error = 0.0;
+      double accu_error = 0.0;
+
+      gpu_blas->validate_gemm(self_check, accu_check, self_error, accu_error);
```

Comment:
unlikely case , but we must ensure atleast 2 runs of this call to compare current and previous results.  how to ensure this ?

Reply:
In src/rvs_blas.cpp, function check_result_consistency() check for comment /* Exit first iteration of self-check as there is no previous result yet ! */ its done here.